wotlk_cat_sim.py

```diff
@@ -363,15 +367,24 @@ def lacerate(self, time):
                 self.player.lacerate_tick * self.lacerate_stacks
                 * (1 + 0.15 * self.player.enrage)
             )
-            self.lacerate_crit_chance = self.player.crit_chance - 0.04
+            self.lacerate_crit_chance = self.player.calc_crit_chance()
```

Comment:
oops, this definitely should remain as crit_chance - 0.02 * master_shapeshifter, and not check form ability, since it snapshots. Will fix tomorrow, here and elsewhere if it's repeated.

Reply:
Resolved.